espnet2/tasks/abs_task.py

```diff
@@ -1265,6 +1273,7 @@ def main_worker(cls, args: argparse.Namespace):
                 ngpu=args.ngpu,
                 log_interval=args.log_interval,
                 write_collected_feats=args.write_collected_feats,
+                skip_stats_npz=args.skip_stats_npz,
```

Comment:
Is this the same purpose as in line 1169 of this file? We set the model to None if we do not want it. Then in collect_stats function, we won't actually do collect_stats.
If so, we can combine this with the existing one and redesign the logic.